tools/webpack.config.dev.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+var version = require('./../package.json').version;
+var path = require('path');
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+
+var rootPath = path.join(__dirname, '..');
+
+function root(p){
+  return path.join(rootPath, p);
+}
+
+var distPath = root('dist');
+
+module.exports = {
+  entry: [
+    'webpack-hot-middleware/client?reload=true',
+    './src/entry'
+  ],
+  output: {
+    path: distPath,
+    publicPath: '/dist',
+    filename: 'bundle.js'
```

Comment:
It makes sense to split webpack config to 2 files: prod and dev.
Production env would include some things like Deduplication or UglifyJsPlugin, look here https://webpack.github.io/docs/list-of-plugins.html#optimize.

Also, it would be wise idea to have bundle.js in [hash].bundle.js for cache busting.